### PR TITLE
rules_scala stewardship update

### PR DESCRIPTION
**Pull request by ittaiz**
Over the past few months @simuons has become a very important part of rules_scala d2d (helping on issues, reviewing PRs, and authoring PRs).  
Seeing as he’s been fulfilling the maintainer roles without being named as one I feel it is only natural to add him as an official maintainer.   

I want to take this opportunity to re-iterate that we are very interested in adding more maintainers (individuals and companies) to make sure rules_scala remains a thriving project and an important part of the bazel ecosystem.   

This close and engaging work by him and @liucijus has gladly brought me to the place where I can leave rules_scala stewardship in their trusted hands and continue to new adventures.  

I want to thank you for your help over this time, I’m very excited about where rules_scala is right now and where it’s going, and I’m sure that we will meet again in the bazel community.

### CODE REVIEW

File: Governance.md
Comment (liucijus) [inline Governance.md:58]: @simuons is this correct?

**Review by liucijus — APPROVED** [review]
@ittaiz thanks for your hard work and dedication!

File: Governance.md
Comment (simuons) [reply Governance.md:58]: Correct. Not sure about the need for `(Pinevicius)` thought.

**Comment by ittaiz** [comment]
@simuons waiting for your approval :)

**Review by simuons — APPROVED** [review]
Thanks @ittaiz

**Pull request merged**
